traits/trait_types.py

```diff
@@ -3424,6 +3428,14 @@ def get_default_value(self):
 
         return (dvt, dv)
 
+    def clone(self, default_value=NoDefaultSpecified, **metadata):
+        """ pass """
```

Comment:
I think it'll be useful to have a better docstring than this or just no docstring at all.

Reply:
Aargh! That was never intended to end up in the finished PR. Will fix.

Reply:
Fixed